### ISSUE TITLE
Swift 5.4.2 framework version required

### ISSUE THREAD

**Issue by johnrogers** [sendbird/sendbird-uikit-ios#12]
```
Incompatible Swift version - framework was built with 5.2.2 (swiftlang-1103.0.32.6 clang-1103.0.32.51) and the local version is 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53).
```

Can you please update the framework, built with Swift 5.2.4? Currently unable to build using Carthage with Xcode 11.5.

**Comment by intoxicated** [comment]
@johnrogers Thanks for the feedback! will get back to you once we update the framework

**Comment by johnrogers** [comment]
Thanks @intoxicated - the same will apply to `sendbird-ios-framework`, too.
Do you think this will be updated by tomorrow? We're currently testing SendBird as a POC and this is a blocker for us.

**Comment by tezpark** [comment]
Hello @johnrogers,

We are trying to fix the Carthage issue. But I can't guarantee release until tomorrow.
To use binary first, you can download and use the release binary directly.

BTW I can't understand that the same problem occurs in `sendbird-ios-framework`.
What happens when you command a `carthage update`?

**Comment by tezpark** [comment]
Hi @johnrogers,

We just modified the binary to the latest Swift version (Swift 5.2.4).
Try the command `carthage update`. 
Let me know if you have any problems again.

Thanks for reporting 👍

**Comment by johnrogers** [comment]
Hi @tezpark , thank you.
It doesn't look like there's a new version of UIKit available - latest is 1.0.7, 5 days ago?
Carthage won't update because there's nothing to update to.

**Comment by johnrogers** [comment]
Any update on this?

**Comment by tezpark** [comment]
HI @johnrogers 
I'm sorry for late reply.
We will update the new version today. And, we'll let you know again when it's released.

**Comment by tezpark** [comment]
@johnrogers, 
I just released v1.0.8. Check it out and let me know if you have any problems!
